bigbluebutton-html5/private/config/settings.yml

```
@@ -34,6 +34,7 @@ public:
       duration: 4000
     remainingTimeThreshold: 30
     remainingTimeAlertThreshold: 1
+    breakoutRoomLimit: 30
```

```suggestion
    breakoutRoomLimit: 8
```

I recommend keeping the old value as the default value. From what I know it was a good fit for the vast majority of use cases. Also a large number like 30 suggests a use case different than the standard "breaking into groups for discussion"

---

I believe that in a University setting many users may want to have a higher limit. However, the most important thing for me is to make it possible (please merge!), not the default number as long as we can adjust it.

---

8 is fine for me.

---

What happens if I set it to 1?

---

MIN_BREAKOUT_ROOMS = 2 constant is still hard-coded, so if you set limit to 1 or a smaller number, you will not be able to create any breakout rooms.

---

So, if we still have a hardcoded bottom cap doesn't make sense to validate the max number to be at least the minimum?

---

Sure, i have added a check for that.